feature-wallet-impl/src/main/java/io/novafoundation/nova/feature_wallet_impl/data/network/blockchain/assets/transfers/orml/OrmlAssetTransfers.kt

```diff
@@ -51,11 +51,11 @@ class OrmlAssetTransfers(
     override val validationSystem: AssetTransfersValidationSystem = defaultValidationSystem()
 
     override suspend fun totalCanDropBelowMinimumBalance(chainAsset: Chain.Asset): Boolean {
-        return false
+        return true
```

Comment:
Seems like it is reasonable to make a default implementation inside AssetTransfers interface to avoid such kind of dublication